purchases/src/test/java/com/revenuecat/purchases/common/backend/BackendGetCustomerCenterConfigTest.kt

```diff
@@ -0,0 +1,315 @@
+package com.revenuecat.purchases.common.backend
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.VerificationResult
+import com.revenuecat.purchases.common.AppConfig
+import com.revenuecat.purchases.common.Backend
+import com.revenuecat.purchases.common.BackendHelper
+import com.revenuecat.purchases.common.Dispatcher
+import com.revenuecat.purchases.common.HTTPClient
+import com.revenuecat.purchases.common.SyncDispatcher
+import com.revenuecat.purchases.common.networking.Endpoint
+import com.revenuecat.purchases.common.networking.HTTPResult
+import com.revenuecat.purchases.common.networking.RCHTTPStatusCodes
+import com.revenuecat.purchases.customercenter.CustomerCenterConfigData
+import com.revenuecat.purchases.customercenter.CustomerCenterConfigData.HelpPath
+import com.revenuecat.purchases.customercenter.CustomerCenterConfigData.Screen
+import com.revenuecat.purchases.customercenter.CustomerCenterConfigData.Screen.ScreenType
+import com.revenuecat.purchases.customercenter.RCColor
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Assert.fail
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.File
+import java.net.URL
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.LinkedBlockingQueue
+import java.util.concurrent.ThreadPoolExecutor
+import java.util.concurrent.TimeUnit
+import kotlin.time.Duration.Companion.seconds
+
+@RunWith(AndroidJUnit4::class)
+class BackendGetCustomerCenterConfigTest {
+
+    private val mockBaseURL = URL("http://mock-api-test.revenuecat.com/")
+    private val MOCK_RESPONSE_FILENAME = "get_customer_center_config_success.json"
+
+    private lateinit var appConfig: AppConfig
+    private lateinit var httpClient: HTTPClient
+
+    private lateinit var backend: Backend
+    private lateinit var asyncBackend: Backend
+
+    private val expectedCustomerCenterConfigData = CustomerCenterConfigData(
```

Comment:
I have never used it, but [Kotest's property test framework](https://kotest.io/docs/proptest/property-based-testing.html) can supposedly help with generating these test fixtures. (Not for this PR of course, just something we could look into at some point.)